app/app.js

```diff
@@ -62,8 +62,10 @@ module.exports = (options) => {
   // serve html5-shiv from node modules
   app.use('/vendor/html5-shiv/', express.static('node_modules/html5shiv/dist/'))
 
-  // serve legacy code from node node modules
+  // serve legacy code from node modules
   app.use('/vendor/govuk_template/', express.static('node_modules/govuk_template_jinja/assets/'))
+  app.use('/vendor/govuk_frontend_toolkit/', express.static('node_modules/govuk_frontend_toolkit/javascripts/govuk/'))
+  app.use('/vendor/jquery/', express.static('node_modules/jquery/dist'))
```

Comment:
# 😱   jQuery !!!

Reply:
Yeah... it's a Frontend Toolkit dependency needed for testing. Rest assured, it will only be included in the review app legacy mode and won't be 🚢